src/clean.cc

```diff
@@ -39,19 +39,19 @@ Cleaner::Cleaner(State* state,
     status_(0) {
```

Comment:
If you want to just rip the bandaid off, you could instead rebase on this commit : https://github.com/splinter-build/splinter/tree/1-std-namespace

Reply:
Interesting. Very interesting. I think I might do that. Are you the one who started the splinter project? I couldn't find any explanation in the code for splinter about who started it and why, other than that it seems to be C++17.

Reply:
Yes. I'm the only contributor of code beyond a PR or two that I've merged.

There's another person on that project that did a bunch of project-management stuff, but I haven't heard from them in quite a while.

Reply:
I update the set of branches in that github repo periodically when ninja updates things.